docker-compose.yml

```diff
@@ -44,6 +44,13 @@ services:
       VELA_ENABLE_SECURE_COOKIE: 'false'
       VELA_REPO_ALLOWLIST: '*'
       VELA_SCHEDULE_ALLOWLIST: '*'
+      VELA_OTEL_EXPORTER_OTLP_ENDPOINT: http://jaeger:4318
```

Comment:
i don't think `VELA_OTEL_EXPORTER_OTLP_ENDPOINT` is used?

Reply:
should be `OTEL_EXPORTER_OTLP_ENDPOINT` for making it easy to use with jaeger all-in-one.

i think we can get away with setting the traces endpoint, let me double check if this is still needed

Reply:
went with `VELA_OTEL_EXPORTER_OTLP_ENDPOINT` with better parsing/validation before using `WithEndpoint`. this makes everyone happy!